readinglist/tests/test_views_article.py

```diff
@@ -53,8 +53,14 @@ def test_mark_by_and_on_are_set_to_none_if_unread_is_true(self):
             'marked_read_by': 'FxOS',
             'marked_read_on': 1234}
         resp = self.app.patch_json(self.url, mark_read, headers=self.headers)
-        self.assertIsNotNone(resp.json['marked_read_by'])
-        self.assertIsNotNone(resp.json['marked_read_on'])
+        body = resp.json
+        self.assertEqual(body['marked_read_by'], mark_read['marked_read_by'])
+        self.assertEqual(body['marked_read_on'], mark_read['marked_read_on'])
+
+        resp = self.app.get(self.url, headers=self.headers)
+        body = resp.json
+        self.assertEqual(body['marked_read_by'], mark_read['marked_read_by'])
+        self.assertEqual(body['marked_read_on'], mark_read['marked_read_on'])
 
```

Comment:
I wanted to check that the ``marked_read_by`` is correctly set to the value provided with the request.